config/locales/en.yml

```
@@ -1207,9 +1211,9 @@ en:
         title: Are you sure you want to delete all question %{question_number}’s routes?
       show:
         any_other_answer:
-          heading: Any other answer
+          heading: If people select any other answer
           set_questions_to_skip: Set questions to skip
-          skip_later: If you need to, you can make these people skip one or more questions later in the form. For example, you might want them to skip route 1’s questions to create 2 different ‘branches’ of questions.
+          skip_later: If you need to, you can make them skip one or more questions that only people on route 1 need to answer.
           will_continue_to: People who select any other answer will continue to question %{next_question_number}.
```

```suggestion
          will_continue_to: People who select any other answer will continue to question %{next_question_number} and through the rest of the form.
```
This was just missing the last part of the sentence - I am hoping this is the ‘question X’s routes’ summary page :)

---

thanks, I've added this and updated the screenshots